neuralmonkey/model/sequence.py

```diff
@@ -119,6 +120,18 @@ def __init__(self,
         if any([esize <= 0 for esize in self.embedding_sizes]):
             raise ValueError("Embedding size must be a positive integer.")
 
+    def save_visualization(self, logdir, projector):
```

Comment:
Tohle by chtělo docstring. Bez googlení bych nezjistil, co je `projector`. Kdybys dělal abstraktní třídu pro embedovací `ModelPart`s (viz další komentář), tak by to chtělo, aby tahle metoda byla v té abstraktní třídě. Pokud ne, tak se napiš TODO.

Zvážil bych ještě přejmenování té metody na nějako `tensorboard_embedding_visualization`.